src/scenes/home/team/team.js

```diff
@@ -38,6 +23,15 @@ class Team extends Component {
       />
     ));
 
+    const executiveStaffs = ExecutiveStaffs.map(executiveStaff => (
+      <ExecutiveStaffCard
```

Comment:
Don't forget when you iterate over an object, React always wants you to create a unique `key` value.

```
key={`${executiveStaff.name} + ${executiveStaff.role}`}
```

should work fine.